src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
```

Comment:
You need to make [closure](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Closures). It's needed to remember the state, and at the next call, it will work with previous calculations.
![image](https://user-images.githubusercontent.com/56752562/218977869-1bcce21f-7de3-4ad9-9c3d-ad7f796e3fb5.png)